src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -88,17 +90,20 @@ public void parse_allFieldsPresent_success() {
                 .build();
 
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND + MATRICULATION_NUMBER_DESC_BOB + GENDER_DESC_BOB,
+                        + BLOCKROOM_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND
+                        + MATRICULATION_NUMBER_DESC_BOB + GENDER_DESC_BOB,
                 new AddCommand(expectedPersonMultipleTags));
 
         // multiple matriculation number - last matriculation number accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB
-                + GENDER_DESC_BOB + TAG_DESC_FRIEND + MATRICULATION_NUMBER_DESC_AMY + MATRICULATION_NUMBER_DESC_BOB,
+                        + BLOCKROOM_DESC_BOB + GENDER_DESC_BOB + TAG_DESC_FRIEND
+                        + MATRICULATION_NUMBER_DESC_AMY + MATRICULATION_NUMBER_DESC_BOB,
```

Comment:
```
assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB
        + BLOCKROOM_DESC_BOB + GENDER_DESC_BOB + TAG_DESC_FRIEND
        + MATRICULATION_NUMBER_DESC_AMY + MATRICULATION_NUMBER_DESC_BOB,
```
Should the indentation be 8 spaces here? https://se-education.org/guides/conventions/java/intermediate.html#layout